Makefile.js

```diff
@@ -938,6 +938,17 @@ target.checkRuleFiles = function() {
             errors++;
         }
 
+        // check parity between rules index file and rules directory
+        const builtInRulesIndexPath = "./lib/built-in-rules-index";
+        const ruleIdsInIndex = require(builtInRulesIndexPath);
+        const ruleEntryFromIndexIsMissing = !(basename in ruleIdsInIndex);
+
+        if (ruleEntryFromIndexIsMissing) {
+            console.error(`Missing rule from index (${builtInRulesIndexPath}.js): ${basename}. If you just added a ` +
+                "new rule then add an entry for it in the previously mentioned file.");
```

Comment:
Small nit/suggestion:

```suggestion
                "new rule then add an entry for it in this file.");
```

Let me know if you think that's too unclear-- if so, we can leave as is. It just felt a little wordy.

Reply:
I was on the fence about it too, so totally fine with changing it. Thank you!